src/products/components/ProductVariantMedia/ProductVariantMedia.tsx

```diff
@@ -38,27 +41,26 @@ const useStyles = makeStyles(
       gridTemplateColumns: "repeat(4, 1fr)"
     }
   }),
-  { name: "ProductVariantImages" }
+  { name: "ProductVariantMedia" }
 );
 
-interface ProductVariantImagesProps {
-  images?: ProductImageFragment[];
+interface ProductVariantMediaProps {
+  media?: ProductMediaFragment[];
   placeholderImage?: string;
   disabled: boolean;
   onImageAdd();
 }
 
-export const ProductVariantImages: React.FC<ProductVariantImagesProps> = props => {
-  const { disabled, images, onImageAdd } = props;
-
-  const classes = useStyles(props);
+export const ProductVariantMedia: React.FC<ProductVariantMediaProps> = props => {
   const intl = useIntl();
+  const classes = useStyles(props);
+  const { disabled, media, onImageAdd } = props;
 
   return (
     <Card>
       <CardTitle
         title={intl.formatMessage({
-          defaultMessage: "Images",
+          defaultMessage: "Media",
```

Comment:
This component also would look bit cleaner with messages moved to defineMessages